src/stories/3-toggle.stories.tsx

```diff
@@ -52,7 +52,10 @@ export const Sizes = () => {
 export const AllState = () => {
   return (
     <Box m={2}>
+      <Label backgroundColor={'dodgerblue'}>foo</Label>
```

Comment:
should you add htmlFor here perhaps? also (minor) you can use bg instead of backgroundColor